<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StockCategories</class>
 <widget class="QMainWindow" name="StockCategories">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stock Items</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="stock_items_table">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>721</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="item_name">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Item Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="id">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="update">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>290</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>290</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="edit">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>90</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QLabel" name="message">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>330</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="additional_information">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Additional Information</string>
    </property>
   </widget>
   <widget class="QComboBox" name="categories">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>250</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="availability">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="item_unit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>210</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Unit</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="price">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>210</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
